local-interchain/cmd/local-ic/start_chain.go

```diff
@@ -40,6 +45,28 @@ var startCmd = &cobra.Command{
 			configPath = filepath.Base(configPath)
 		}
 
+		var config *types.Config
+		var err error
+
+		if strings.HasPrefix(configPath, "http") {
+			config, err = interchain.LoadConfigFromURL(configPath)
+			if err != nil {
+				panic(err)
+			}
+
+			// last part of the URL to be the test name
+			configPath = configPath[strings.LastIndex(configPath, "/")+1:]
```

Comment:
`configPath` can become any arbitrary string depending on what URL the user passes in, is this okay? is it solely used for creating the name of the running test case?

Reply:
yes, it is only used as the test name (and technically nothing here is also okay as other data is appended)

else you get a super long testname for gist which takes up the an entire terminal line. really messy